full-service/src/json_rpc/v2/api/request.rs

```diff
@@ -53,6 +53,16 @@ pub enum JsonCommandRequest {
         max_spendable_value: Option<String>,
         comment: Option<String>,
     },
+    build_burn_transaction {
+        account_id: String,
+        amount: Amount,
+        redemption_memo: Option<String>,
+        input_txo_ids: Option<Vec<String>>,
+        fee_value: Option<String>,
+        fee_token_id: Option<String>,
```

Comment:
Is this for a world where the fee is different than the token we are transacting in?

Reply:
In the eventual future of mixed token transactions, yes. At this exact moment, no.